srcpkgs/gdm/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'gdm'
 pkgname=gdm
 version=3.38.2.1
-revision=2
+revision=3
 build_helper="gir"
 build_style=meson
 configure_args="
  -Ddefault-pam-config=arch -Dat-spi-registryd-dir=/usr/libexec
  -Dtcp-wrappers=false -Dscreenshot-dir=/var/lib/gdm/greeter
- -Dplymouth=disabled -Dxauth-dir=/run/gdm -Dpid-file=/run/gdm/gdm.pid
+ -Dplymouth=enabled -Dxauth-dir=/run/gdm -Dpid-file=/run/gdm/gdm.pid
  -Dsystemd-journal=false -Dinitial-vt=7 -Dwayland-support=true
  -Dselinux=disabled -Dlibaudit=disabled
  -Dsystemdsystemunitdir=/tmp -Dsystemduserunitdir=/tmp"
 hostmakedepends="dconf gettext itstool pkg-config"
 makedepends="accountsservice-devel elogind-devel gettext-devel glib-devel
- iso-codes libSM-devel libcanberra-devel nss-devel pam-devel upower-devel"
+ iso-codes libSM-devel libcanberra-devel nss-devel pam-devel upower-devel
+ plymouth-devel"
```

Comment:
So it checks for the plymouth library at configure time (I could find this line), but doesn't link against it? At least I couldn't see anything about it in the CI log...

Just wondering what it is that it does with the library, and if we need any sort of warning about plymouth library revbump or such.

Reply:
It does absolutely nothing with the library, yet it requires `ply-boot-client.pc` to be present at configure time. Code just spawns `plymouth` binary and that's all.

The quirky logic comes from a patch written by Fedora devs 8 years ago. My best guess is that it was upstreamed without anyone knowing what it really did.